Core/BFFT/test.hpp

```diff
@@ -7,15 +7,23 @@
 *
 **/
 
-#ifndef INC_MYMAIN_HPP_
-#define INC_MYMAIN_HPP_
+#ifndef TEST_H
+#define TEST_H
+
+typedef enum
+{
+   TEST_INIT,
+   TEST_EXECUTE,
+   TEST_ERROR,
+} TEST_STATES;
```

Comment:
According to the coding conventions 

eTest_States_t

also use enum class over enum if possible